packages/cloning-decorator/tsconfig.base.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compileOnSave": false,
+    "compilerOptions": {
+      "rootDir": ".",
+      "sourceMap": true,
+      "declaration": false,
+      "moduleResolution": "node",
+      "emitDecoratorMetadata": true,
+      "experimentalDecorators": true,
+      "importHelpers": true,
+      "target": "es2015",
+      "module": "esnext",
+      "lib": ["es2020", "dom"],
+      "skipLibCheck": true,
+      "skipDefaultLibCheck": true,
+      "baseUrl": ".",
+      "paths": {
+        "@datev/cloning-decorator": ["packages/cloning-decorator/src/index.ts"]
```

Comment:
Review this reference and remove the reference to datev please.

Reply:
Thank you @hugoib 
That has been corrected